web/client/components/app/StandardApp.jsx

```diff
@@ -10,7 +10,7 @@ const {Provider} = require('react-redux');
 const PropTypes = require('prop-types');
 const dragDropContext = require('react-dnd').DragDropContext;
 const html5Backend = require('react-dnd-html5-backend');
-const proj4 = require('proj4').default;
+const proj4 = require('@geosolutions/proj4').default;
```

Comment:
Maybe we can also have an alias in webpack config files so that any require('proj4') will indeed use @geosolutions/proj4. The changes are ok, I would add the alias as an additional fallback.

Reply:
sounds good